docs/pipeline/profile.md

```diff
@@ -15,18 +15,18 @@ In the figure below, it is indicated that the values inGoodread.details are cont
 ![profiling-config](https://i.imgur.com/5ra1ubM.png)
 Profiling parameters configure the discovery of candidate keys and the discovery of inclusion dependencies. 
```

Comment:
Have the text start on a newline, not straight after the image

Reply:
Done!